generic-styles/_content-common.less

```diff
@@ -0,0 +1,512 @@
+// Constants from bootstrap/less/variables.less
```

Comment:
do we need this file at all? we only have baked content in rex right?

Reply:
From the discussions Unified folks (aka "me") would be fixing any problems with the generic styles. Since there is no way to preview the rendered HTML in `cnx-recipes` I put the code in here.